crates/solver/src/settlement_post_processing/optimize_buffer_usage.rs

```diff
@@ -41,6 +53,23 @@ fn is_only_selling_trusted_tokens(
         .any(|order| !market_makable_token_list.contains(&order.data.sell_token))
 }
 
+fn some_stable_to_stable_trade(settlement: &Settlement) -> bool {
+    let stable_coins = [
+        H160(hex_literal::hex!(
+            "A0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48" // USDC
+        )),
+        H160(hex_literal::hex!(
+            "6B175474E89094C44Da98b954EedeAC495271d0F" // DAI
+        )),
+        H160(hex_literal::hex!(
+            "dAC17F958D2ee523a2206206994597C13D831ec7" // USDT
+        )),
+    ];
+    settlement.traded_orders().any(|o| {
+        stable_coins.contains(&o.data.sell_token) && stable_coins.contains(&o.data.buy_token)
+    })
```

Comment:
Can we additionally add a condition that:
1. It involves stable-stable coins
2. The internalized interaction would trade at more than 1:1?

The 2. condition allows us to internalize in the general case to keep on saving gas money. Stable-stables are a lot of our trades.

Reply:
According to [this](https://cowservices.slack.com/archives/C0375NV72SC/p1686552438506019?thread_ts=1686315504.555289&cid=C0375NV72SC) the benefits of the second condition will be minimal, so I would say its not worth implementing.